src/services/iiif-parser.js

```diff
@@ -118,15 +118,23 @@ export function getTracks() {
  * @param {Object} label
  */
 export function getLabelValue(label) {
+  let decodeHTML = (lableText) => {
+    return lableText
+      .replace(/&amp;/g, '&')
+      .replace(/&lt;/g, '<')
+      .replace(/&gt;/g, '>')
+      .replace(/&quot;/g, '"')
+      .replace(/&apos;/g, "'");
+  };
```

Comment:
This is just for rendering the HTML string not an HTML element, right?  HTML is only allowed in limited places in a manifest.  (https://iiif.io/api/presentation/3.0/#45-html-markup-in-property-values)

Reply:
Yes this is for HTML strings. For an example; if you go to the GitHub demo page and select `Lunchroom manners` manifest from IU collection, `&` is replaced by `&amp;` in the structure. This is what I wanted to fix. We use the same thing in SME.